bin/garden-build

```diff
@@ -114,12 +114,7 @@ gpg --batch --no-default-keyring --keyring "$keyring" --import "$keyringPlain"
 	[ -n "$fullfeatures" ] && configArgs+=( --features "$fullfeatures" )
 
 	# TODO fix for the pipelines, paths probably won't match
```

Comment:
Not sure if this TODO refers to the following deleted lines.

Reply:
Please delete this TODO comment. If this line is still relevant we should open an issue.